code/src/App.js

```diff
@@ -1,9 +1,21 @@
-import React from 'react'
+import React from 'react';
+import { Provider } from 'react-redux';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+
+import { labyrinthSlice } from 'reducers/labyrinthSlice';
+import { Game } from 'components/Game';
+
+const reducer = combineReducers({
+	labyrinthSlice: labyrinthSlice.reducer,
+	
```

Comment:
Do you use your ui Slice? I can't see that it has been imported anywhere (or maybe I have overlooked it?).